### PR TITLE
Psv 44 upload

### PR DESCRIPTION
**Pull request by appdoo18**
#### JIRA LINK ####

- Please include JIRA ticket link here if any

- https://patent-visualization.atlassian.net/jira/software/projects/PSV/boards/1/backlog?selectedIssue=PSV-44

#### CHANGES ####

- Please list the relevant changes in the PR

-Skeleton set up for patents to be uploaded
-Text field to enter protein name to be used when storing in database and S3 bucket
-Ability to browse for a local file

#### DETAILS ####

- Any relevant details/references based on this changes

- Still need to work on having the patent uploaded to the S3 bucket and database.
- DO NOT APPROVE YET PLEASE, just wanted to demonstrate the skeleton

#### MANDATORY GIF ####

PR will **NOT** be aproved if no GIF is included 😄
Include small/social giphy inside the parens **()** https://giphy.com/

giphy 1

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
```
Comment (blam92) [inline .husky/pre-commit:1]: make sure to do git pull from `main` and `npm install`. This should not be deleted

File: src/components/PatentUpload/PatentUpload.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.protein-container {
+    margin: auto;
+}
+
+.protein-container_input {
```
Comment (blam92) [inline src/components/PatentUpload/PatentUpload.css:5]: Use __ double underscore to separate block from element`.protein-container__input`

File: src/components/PatentUpload/PatentUpload.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    margin: auto;
+    width: 30%;
+}
+
+.upload-container_input {
```
Comment (blam92) [inline src/components/PatentUpload/PatentUpload.css:10]: Use __ double underscore to separate block from element

File: src/components/PatentUpload/PatentUpload.js [hunk 4]
```diff
@@ -1,4 +1,96 @@
...
+                <Form.Item
+                    className="protein-container_input"
+                    label={StringManager.get('typeProtein')}
+                    name="protein"
+                    rules={[{ required: false }]}
+                >
+                    <Input />
+                </Form.Item>
+            </Form>
+            <input
```
Comment (blam92) [inline src/components/PatentUpload/PatentUpload.js:58]: Ideally we should not need to create any html element and leverage ant design. See some options here:
https://ant.design/components/input/

File: src/components/PatentUpload/PatentUpload.js [hunk 5]
```diff
@@ -1,4 +1,96 @@
...
+                    pdfFile: this.upload.files[0],
+                    fileName: this.upload.files[0].name
+                })
+                }
+            />
+            <input
+            className="upload-container_input" 
+            value={this.state.fileName} placeholder="Select file" 
+            />
+            <button
```
Comment (blam92) [inline src/components/PatentUpload/PatentUpload.js:75]: Same as input, look for antd components

**Review by blam92 — COMMENTED** [review]
Looking good! I know it's a skeleton just left some comments for future edits. 
Please look into using Antd upload component https://ant.design/components/upload/
They look nice and you can leverage a lot of their logic 😉  (you can look into their code by clicking on the small icons below the component to open it on a sandbox)

**Comment by appdoo18** [comment]
Redoing with Santi's comments

**Pull request closed without merging**